oonf-olsrd2/Config.in

```diff
@@ -44,4 +44,16 @@
 			The MPR plugin reduce the routing graph to limit the overhead of the OLSRv2 protocol
 		default n
 
+	config OONF_OLSRV2_LAN
+		bool "New config option for Locally attached entries"
```

Comment:
Why the upper case L for Locally?

Reply:
its copy paste from https://github.com/OLSR/OONF/blob/master/openwrt/oonf-olsrd2-git/Config.in#L54

Reply:
hm, at little weird, but ok